### PR TITLE
[12.0] Add ir_filters_multi_company

### PR DESCRIPTION
**Pull request by sbejaoui**

### CODE REVIEW

File: ir_filters_multi_company/models/ir_filters.py
Comment (hhgabelgaard) [inline ir_filters_multi_company/models/ir_filters.py]: Change to ir.filter

Comment (sbejaoui) [reply ir_filters_multi_company/models/ir_filters.py]:
```suggestion
            'ir.filters'
```

**Comment by elvise** [comment]
hi, any good news for merge?
how can i help for this task...?

**Review by gva-acsone — APPROVED** [review]
LGTM functional test

**Review by hhgabelgaard — APPROVED** [review]

**Comment by sbejaoui** [comment]
@OCA/core-maintainers , This PR is ready to merge

**Comment by simahawk** [comment]
@sbejaoui thanks for your work! Here are few questions/demands.
Any reason not to ping the PSC team?
Can you squash (the bot is not able to squash yet) and include (always) the module name in the commit message pls?

**Comment by GSLabIt** [comment]
Any reason to keep this on hold?

**Comment by dreispt** [comment]
@simahawk ?

**Comment by sbejaoui** [comment]
> @sbejaoui thanks for your work! Here are few questions/demands.
> Any reason not to ping the PSC team?
> Can you squash (the bot is not able to squash yet) and include (always) the module name in the commit message pls?

@simahawk, done

**Pull request merged**
